docs/application/native/guides/personal/data-sync.md

```diff
@@ -286,7 +293,9 @@ To inform a service application of the time interval at which to operate a sync
 
```

Comment:
line 273: The separated lines are merged as one line.
line 377: The separated lines are merged as one line.

Reply:
done